startupscript/vertex-ai-user-managed-notebook/post-startup.sh

```diff
@@ -752,6 +754,27 @@ EOF
 chown ${LOGIN_USER}:${LOGIN_USER} "${USER_BASHRC}"
 chown ${LOGIN_USER}:${LOGIN_USER} "${USER_BASH_PROFILE}"
 
+####################################
+# Run a user provided startup script
+####################################
+
+# If the user has provided a startup script, run it after workbench setup,
+# but before restarting Jupyter and running tests.
+
+readonly USER_STARTUP_SCRIPT="$(get_metadata_value "instance/attributes/terra-user-startup-script")"
+if [[ -n "${USER_STARTUP_SCRIPT}" ]]; then
+  emit "Running user provided startup script..."
+  readonly USER_STARTUP_SCRIPT_FILE="${USER_TERRA_CONFIG_DIR}/user-startup-script.sh"
+
+  # Copy the user's startup script to the user's .terra directory
+  curl -L "${USER_STARTUP_SCRIPT}" >"${USER_STARTUP_SCRIPT_FILE}"
```

Comment:
Is there a preference against `-o <filename>` or `--output <filename>`?

If there is, then add a space after the redirect ">" and before the filename.

Reply:
No, I'll make use of the -o flag if thats better.